### ISSUE TITLE
Import from csv/xls chokes on large number of articles

### ISSUE THREAD

**Issue by fernplant** [foodcoops/foodsoft#545]
One of my suppliers (Frontier Co-op) has just under 8000 items in its catalog that I attempted to add via csv upload. This is clearly more articles than what the software intended because it grinds to a halt when attempted

**Comment by wvengen** [comment]
That's right. I have yet to see a foodcoop that actually wants to sell all items in the catalog, but you may be the first to sell about 8k products. Usually, when there are this many products, a foodcoop wants to make a selection. That is where the [shared database](https://github.com/foodcoops/foodsoft/wiki/Shared-database) feature comes in, but that requires setting up [sharedlists](https://github.com/foodcoops/sharedlists).

**Comment by wvengen** [comment]
Do you want to offer all 8000 items as part of an order?
